test/public/index.js

```diff
@@ -11,8 +11,8 @@
  * or submit itself to any jurisdiction.
  */
 
-const LogsSuite = require('./logs.test');
+const LogsSuite = require('./logs');
 
-module.exports = () => {
+module.exports = function () {
```

Comment:
Any reason not to use an arrow function here?

Reply:
Not really, we were trying to pass arguments to the tests but to no avail. Will turn it back to an arrow function.